projects/cashmere-examples/src/lib/drawer-basic/drawer-basic-example.component.scss

```diff
@@ -1,4 +1,4 @@
 .hc-drawer {
-    background-color: #6e6f71;
-    color: #b9babb;
+    background-color: #384655;
```

Comment:
any reason not to use color variables here instead of a hardcoded hex?

Reply:
Yeah - those are in examples, and it seemed like we weren't pulling sass variables into any other examples.  I'm not even sure what the path would be to do that.  Any idea?

Reply:
Should be able to add this to the top of the file I think:
```
@import "~@healthcatalyst/cashmere/scss/colors";
```